builds/jazz-build-module/environment-deployment-metadata-loader.groovy

```diff
@@ -96,22 +96,27 @@ def getEnvironmentInfo() {
 	}
 }
 
-def getEnvDeploymentDecscriptor() {
+def getEnvDeploymentDecscriptor(env_logical_id) {
```

Comment:
No need to pass  the env_logical_id here, instead we can use the g_environment_logical_id

Reply:
updated.